NitroxClient/MonoBehaviours/Gui/Chat/PlayerChat.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using NitroxClient.GameLogic.ChatUI;
+using NitroxModel.Core;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace NitroxClient.MonoBehaviours.Gui.Chat
+{
+    public class PlayerChat : uGUI_InputGroup
+    {
+        private const int LINE_CHAR_LIMIT = 255;
+        private const int MESSAGE_LIMIT = 64;
```

Comment:
Rename to indicate it refers to multiple message limit not just 'a' message limit.

```suggestion
        private const int MESSAGES_LIMIT = 64;
```